src/FlyWithLua.cpp

```diff
@@ -6555,6 +6560,22 @@ void ResetLuaEngine()
     }
 }
 
+bool ReadPrefFile()
```

Comment:
Suggestion: instead of a collection of standalone functions and global variables I suggest creating a single object, which will encapsulate all the logic related to configuration. For example, it would expose two methods for the developer mode flag:

1. `IsDeveloperMode()` --- returns current state of delevoper mode based on ini config, menu state and whatever else. For anyone wanting to condition some logic on developer mode this method will be the source of truth.
2. `SetDeveloperMode(bool flag)` --- switches developer mode and updates config on disk, menu state and whatever internal state may exist.

The goal is to provide an easy to use abstraction on top of config file and menu flags, so that we don't have to worry about global flags, menu and ini files going out of sync in any other place in the code.

Reply:
Like the idea but not comfortable enough with my coding skils to do that without breaking something.

Reply:
I think you're underestimating your coding skills :) As well as I think that in this way it would be much easier not to break anything. But ultimately it's your call :)